src/Components/review.jsx

```diff
@@ -39,8 +47,21 @@ export default function Review() {
         Posted by: {review.owner} at {review.created_at}
         <br />
         <br />
+        {comments.length === 0 ? (
+          "There are no Comments"
+        ) : (
+          <ul>
+            Comments: <br /> <br />
+            {comments.map((obj) => {
+              return (
+                <li key={obj.comment_id}>
+                  {obj.body} <br />
+                  Posted by: {obj.author} at {obj.created_at} <br /> <br />
+                </li>
+              );
+            })}
+          </ul>
+        )}
```

Comment:
There is a lot happening in this single component. Comments can be extracted out into a separate component